src/parser/jobs/sge/modules/Gauge.tsx

```diff
@@ -177,4 +191,65 @@ export class Gauge extends CoreGauge {
+		const lostAddersgall = Math.floor(addersgallExpirationTime / ADDERSGALL_TIME_REQUIRED)
+
+		if (lostAddersgall > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: this.data.actions.KERACHOLE.icon,
+				content: <Trans id="sge.gauge.suggestions.lost-addersgall.content">
+					You lost Addersgall due to capping the gauge and letting the timer stop. Your Addersgall actions are your primary healing and mitigation tools, as well as contributing to your MP recovery, so you should try to use another one before regaining your third stack.
+				</Trans>,
+				severity: SEVERITY.MEDIUM,
+				why: <Trans id="sge.gauge.suggestions.lost-addersgall.why">
+					<Plural value={lostAddersgall} one="# Addersgall stack was" other="# Addersgall stacks were"/> lost to timer inactivity.
+				</Trans>,
+			}))
+		}
+
+		const rhizomataLostStacks = Math.floor(this.rhizomataLoss / ADDERSGALL_TIME_REQUIRED)
+		if (rhizomataLostStacks > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: this.data.actions.KERACHOLE.icon,
+				content: <Trans id="sge.gauge.suggestions.lost-to-rhizomata.content">
+					You lost Addersgall due to capping the gauge with <DataLink action="RHIZOMATA" />, which wastes the time already spent charging the third stack. Try to use <DataLink showIcon={false} action="RHIZOMATA" /> when you are at one stack or less to keep from losing timer progress.
+				</Trans>,
+				severity: SEVERITY.MINOR,
+				why: <Trans id="sge.gauge.suggestions.lost-to-rhizomata.why">
+					<Plural value={rhizomataLostStacks} one="# Addersgall stack was" other="# Addersgall stacks were"/> lost to capping the gauge with <DataLink showIcon={false} action="RHIZOMATA" />.
+				</Trans>,
+			}))
+		}
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.TOXIKON_II.icon,
+			content: <Trans id="sge.gauge.suggestions.addersting-overcap.content">
+				<DataLink action="TOXIKON_II" /> is a useful movement and weaving tool, and does the same single-target DPS as <DataLink action="DOSIS_III" />. Try not to waste them by breaking a fourth <DataLink status="EUKRASIAN_DIAGNOSIS" /> shield before using an Addersting stack.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			value: this.adderstingGauge.overCap,
+			why: <Trans id="sge.gauge.suggestions.addersting-overcap.why">
+				<Plural value={this.adderstingGauge.overCap} one="# Addersting stack" other="# Addersting stacks"/> lost due to overcap.
+			</Trans>,
+		}))
+
+		// Calculate the number of possible addersgall stacks the player could have used: Initial max stacks, plus the maximum possible stacks if they kept the timer rolling the entire fight (aside from first-use leniency), and maximum usage of Rhizomata
+		const potentialAddersgalls = ADDERSGALL_MAX_STACKS + Math.floor((this.parser.pull.duration - addersgallLeniency)  / ADDERSGALL_TIME_REQUIRED) + this.cooldownDowntime.calculateMaxUsages({cooldowns: [this.data.actions.RHIZOMATA]})
```

Comment:
Fights with a forced downtime window >1 minute aren't totally uncommon (e1s, e8s, most ultimates)... should we make some effort to consider that downtime when calculating maximum possible addersgall, similar to what we do with forceGainUtaWindows for the overcap suggestion?

For that matter, it looks like this will result in a `used / possible` value that doesn't match the number lost in the suggestion if there's any overcap during downtime, which seems like a poor UX.

Reply:
Revised the potential stack calculation such that it should take the uta info into account now, by way of summing the actual amount generated with the stacks that were lost to timer downtime and the unused uses of Rhizomata.